packages/cron/src/lib/email.js

```diff
@@ -0,0 +1,89 @@
+import debug from 'debug'
+
+export const emailType = {
+  Used75PercentStorage: 'Used75PercentStorage',
+  Used80PercentStorage: 'Used80PercentStorage',
+  Used85PercentStorage: 'Used85PercentStorage',
+  Used90PercentStorage: 'Used90PercentStorage',
+  UsedOver100PercentStorage: 'UsedOver100PercentStorage'
```

Comment:
In my mind, all of these are implicitly "over" whatever the number is (or more precisely, greater than or equal to the number). So in my head the 100% one doesn't need an explicit "over". (I get that it's the top one, but in theory we could add ones for 110% and 120% as well (depending on whether/how/when we restrict them), so it's not necessarily the top one.)

Reply:
That makes sense. Have removed the "over"